web/src/stream_ui_updates.js

```diff
@@ -119,6 +120,7 @@ export function update_regular_sub_settings(sub) {
     if (sub.subscribed) {
         $settings.find(".personal_settings").addClass("in");
         $settings.find(".stream-email-box").show();
+        stream_events.update_property(sub.stream_id, "push_notifications", sub.push_notifications);
```

Comment:
I think we can also remove this, if we set `push_notifications` inside `mark_susbcribed` just like we set `color` of the stream.